erp_facade_service_openApi.yml

```diff
@@ -0,0 +1,210 @@
+openapi: 3.0.1
+info:
+  title: UKHO ERP Facade Service API
+  description: |
+     UKHO ERP Facade Service shall process S57 and S100 Content publish event received from Enterprise Event Service.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+servers:
+  - url: https://erpfacade-dev-api.azurewebsites.net
+    description: Dev URL
+paths:
+  /api/v2/webhook:
+    options:
+      summary: "Handle CloudEvent webhook validation"
+      tags:
+        - "ERP Facade"
+      description: "Handles CloudEvent webhook validation request."
+      responses:
+        "200":
+          description: OK
+        "401":
+          description: Unauthorised - either you have not provided valid token, or your token is not recognised.
+        "403":
+          description: Forbidden - you have no permission to use this API  
+        "500":
+          description: Internal Server Error.
+    post:
+      summary: "Webhook to receive events from Enterprise Event Service."
+      security:
+        - jwtBearerAuth: []
+      tags:
+        - "ERP Facade"
+      operationId: "eesWebhook"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/eventSchema'
+      responses:
```

Comment:
Please can we also have HTTP status 400 - Bad request response here? E.g. Unknown event type.

Reply:
Thank you Vijendra for review comments. This PR got merged now. We will update this changes in separate a branch and PR.